packages/docs/components/SideNav/SideNavLogo/SideNavLogo.tsx

```diff
@@ -1,13 +1,16 @@
 import { Small } from '@bigcommerce/big-design';
+import Image from 'next/image';
 import Link from 'next/link';
 import React from 'react';
 
+import Logo from '../../../public/logo-with-text.svg';
```

Comment:
🍹 can this use `~/`?

Reply:
it'll be a hassle. either have to combat eslint or do some tsconfig `paths` magic.